### ISSUE TITLE
Environment non ascii content

### ISSUE THREAD

**Issue by dgoujard** [phusion/baseimage-docker#96]
Hi,

I would like use an non ascii character in environment variable but i have an issue with my_init

```
docker run -e test="ça marche" 231ab15480b9
*** Killing all processes...
Traceback (most recent call last):
  File "/sbin/my_init", line 312, in <module>
    main(args)
  File "/sbin/my_init", line 232, in main
    export_envvars()
  File "/sbin/my_init", line 78, in export_envvars
    f.write(value)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-1: ordinal not in range(128)
```

It's not a big deal, i would fix it myself if i knew python but i think it's this file https://github.com/phusion/baseimage-docker/blob/master/image/my_init#L78 and this stackoverflow related topic http://stackoverflow.com/questions/5141559/unicodeencodeerror-ascii-codec-cant-encode-character-u-xef-in-position-0

**Comment by etopian** [comment]
Bump. It's been a year.
<code>
import codecs
with codecs.open(file name,'r',encoding='utf8') as f:
    text = f.read()
# process Unicode text
with codecs.open(file name,'w',encoding='utf8') as f:
    f.write(text)
</code>

**Comment by etopian** [comment]
Ok this is actually a locale problem which are explained in detailed here and have to do with the fact that upon bootup, the locales are not properly defined which causes Python to implicitly convert things which causes problems: 

http://stackoverflow.com/questions/2596714/why-does-python-print-unicode-characters-when-the-default-encoding-is-ascii/21968640#21968640

The quick fix is to simply export the locale variable so that Python knows which locale to use.

the flag is  -e 'LC_ALL=en_US.UTF-8'

ex
 docker run -e 'TEST=täst.de' -e 'PYTHONIOENCODING=utf-8' -e 'LC_ALL=en_US.UTF-8' myimage

**Comment by koolay** [comment]
I have the same problem

**Comment by Theaxiom** [comment]
This should now be fixed by #336
